CONTENTS.md

```diff
@@ -363,11 +365,15 @@ Checks the status of the staging workflow bots.
 * Sources: [status.py](status.py)
 * Documentation: --
 * Package: openSUSE-release-tools
-* Usage: ???
+* Usage: obsolete?
```

Comment:
Maybe we should point here that this looks obsolete because it checks for the status of some bots that are already retired.